pkg/cli/proxy_get.go

```diff
@@ -72,7 +73,7 @@ func getPortForwarder(clientSet kubernetes.Interface, config *rest.Config, names
 	if err != nil {
 		return nil, fmt.Errorf("error getting pod %s/%s", namespace, podName)
 	}
-	if !mesh.ProxyLabelExists(*pod) {
```

Comment:
This looks like a functional change; I'd prefer for functional changes to be in a separate PR from linter appeasement